### PR TITLE
Use new Loader for success screens

### PR DESCRIPTION
**Pull request by sisou**

### CODE REVIEW

File: src/views/SignTransactionSuccess.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-            <Success
-                text="Your transaction[br]is ready!"
-                buttonText="Send now"
-                :disabled="!isTxPrepared"
-                @continue="done"
-            />
+            <Loader state="success">
+                <template slot="success">
+                    <div class="success nq-icon"></div>
+                    <h1 class="title nq-h1">Sending your<br>transaction now.</h1>
```
Comment (Bettelstab) [inline src/views/SignTransactionSuccess.vue]: I'm not quite happy with stating it is sent now, while it isn't before returning to Safe.

Comment (sisou) [reply src/views/SignTransactionSuccess.vue]: Especially with the new pico consensus, we could just as well send it here, just like during checkout.

**Comment by sisou** [comment]
Rebased on `master` to fix merge conflicts.

**Comment by Bettelstab** [comment]
Please rename Loader to e.g. StatusIndicator :)

**Comment by sisou** [comment]
#135 also changes the Loader component quite a bit. I'd rather wait with the renaming until after both are merged. I created #142 to not forget about it.

**Pull request merged**